### PR TITLE
chore: Update Communications Handbook

### PR DESCRIPTION
**Pull request by rawkode**
This is based on what I experienced during the v1.17 release.

Feedback from @castrojo and @onlydole appreciated

### CODE REVIEW

File: release-team/role-handbooks/communications/README.md [hunk 1]
```diff
@@ -97,13 +94,11 @@ No activities scheduled
...
+- This is release week!
+  - Keep CNCF in the loop with any potential delays to the release
+- Publish release blog
 
 ### Week 13
 
 - Release retrospective participation
-- Follow-up support with the media
-- Obtain media coverage report from the CNCF team
+- Update this document!
```
Comment (onlydole) [inline release-team/role-handbooks/communications/README.md]: YES! 👏 😄

File: release-team/role-handbooks/communications/README.md [hunk 2]
```diff
@@ -97,13 +94,11 @@ No activities scheduled
 
 ### Week 12
 
-- Release day
-- Continue managing release timing in case of delay
-- Publish the blog
-- Help identify release team and community members for media interview requests
+- This is release week!
+  - Keep CNCF in the loop with any potential delays to the release
```
Comment (onlydole) [inline release-team/role-handbooks/communications/README.md]: Nit: The indenting for this item should be moved left.

File: release-team/role-handbooks/communications/README.md [hunk 3]
```diff
@@ -34,60 +34,57 @@ The expected time investment for both leads and shadows are as follows:
...
-- Continue analysis of planned enhancements
-- Begin attending SIG meetings where enhancements will be delivered from
+- This is enhancements freeze. The fun begins next week
 
 ### Week 5
 
-- Finalize release theme and messaging
-- Being working with the CNCF team to determine 5 blog series and schedule webinar
+- Work with enhancements and release note leads to determine which deliverables are most note worthy
+  - Approach SIG Chairs for said items and request a couple of paragraphs on each deliverable
```
Comment (onlydole) [inline release-team/role-handbooks/communications/README.md]: Nit: The indenting for this item should be moved left.

File: release-team/role-handbooks/communications/README.md [hunk 4]
```diff
@@ -34,60 +34,57 @@ The expected time investment for both leads and shadows are as follows:
...
-- Begin organizing content from SIGs into blog topics, and high-level deliverables for the release
-- Begin attending release team meetings
-- Finalize lead and shadow roles
+- N/A
 
 ### Week 3
 
-- Begin analyzing the release notes draft
 - Work with the enhancements lead to understand what might be delivered
+  - You should get access to the enhancements tracking sheet. You should check this regularly
```
Comment (onlydole) [inline release-team/role-handbooks/communications/README.md]: Nit: The indenting for this item should be moved left.

File: release-team/role-handbooks/communications/README.md [hunk 5]
```diff
@@ -34,60 +34,57 @@ The expected time investment for both leads and shadows are as follows:
...
 ### Week 1
 
-- Former communications coordinator helps identify new leads and shadows
-- Help finalize communications milestones in the release timeline
-- Look for event CFP deadlines to submit release to appropriate event; submit accordingly
+- Select your shadows
+- Ensure yourself and your shadows are entered into the release contact sheet
+- Ensure yourself and your shadows are on the release team meeting invites
+- Check if there are any holidays or events (KubeCon!) that occur during this release
+  - This can/will affect your communication with the CNCF and SIG Chairs. Plan accordingly
```
Comment (onlydole) [inline release-team/role-handbooks/communications/README.md]: Nit: The indenting for this item should be moved left.

**Comment by rawkode** [comment]
@onlydole Those were all double indented on purpose to signify followup remarks to the above item.

Do you think this isn't clear enough?

**Comment by onlydole** [comment]
Ah! Got it, that makes sense to me 👍🏼 it wasn't initially clear to me, but that's only because I haven't see it used too often!

**Pull request merged**